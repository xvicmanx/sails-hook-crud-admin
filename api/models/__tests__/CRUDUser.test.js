import model from '../CRUDUser';

describe('api::models::CRUDUser', () => {
  test('The model is defined properly', () => {
    expect(model).toMatchSnapshot();
  });
});
