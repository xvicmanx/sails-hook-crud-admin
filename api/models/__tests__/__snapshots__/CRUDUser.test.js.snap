// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`api::models::CRUDUser The model is defined properly 1`] = `
Object {
  "attributes": Object {
    "groups": Object {
      "collection": "crudgroup",
    },
    "password": Object {
      "required": true,
      "type": "string",
    },
    "username": Object {
      "required": true,
      "type": "string",
    },
  },
  "beforeCreate": [Function],
  "beforeUpdate": [Function],
  "customToJSON": [Function],
  "verifyPassword": [Function],
}
`;
